src/mmw/js/src/modeling/models.js

```diff
@@ -40,6 +40,16 @@ var Tr55TaskModel = coreModels.TaskModel.extend({
     )
 });
 
+var RwdTaskModel = coreModels.TaskModel.extend({
+    defaults: _.extend(
+        {
+            taskName: 'rwd',
+            taskType: 'modeling'
+        },
+        coreModels.TaskModel.prototype.defaults
+    )
+});
```

Comment:
Looks like this model is not exported. It's also defined in `src/mmw/js/src/draw/models.js`.